ci/checks/enforce-correct-shebangs.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail -u
+
+if grep -r -e '^#!/usr/bin/env python$' -e '^#!/usr/bin/env python2$' git_machete; then
+  echo "Ambigouos python shebang, please declare shebangs that point to python3:  #!/usr/bin/env python3"
```

Comment:
```suggestion
  echo "Ambiguous python shebang, please declare shebangs that point to python3:  #!/usr/bin/env python3"
```
;)

`-ouos` suffix is non-existent